### ISSUE TITLE
[Feat] : Improving the design of the select box

### ISSUE THREAD

**Issue by Urmilasc** [wikicurricula-uy/wikicurricula-boilerplate#25]
Improvements to be done in terms of css, emdedding some hover effect, border to the select box.

Design it on your own way !!

Here is picture for refrence  : - 
![Screenshot 2023-10-07 093142](https://github.com/wikicurricula-uy/wikicurricula-boilerplate/assets/97338915/3c02f572-1cd6-46da-bdf3-204ac877a00c)

**Comment by lutfiaomarr** [comment]
hy @Urmilasc do i need to be assigned to work on this issue?

**Comment by Urmilasc** [comment]
No, I think as per mentioned in the recently shared docs in telegram group , you can start working on issue just by writing that you are working on it
